plugins/BEdita/Core/src/Model/Validation/ObjectTypesValidator.php

```diff
@@ -42,12 +42,14 @@ public function __construct()
         $this
             ->requirePresence('name', 'create')
             ->notEmpty('name')
+            ->regex('name', '/^([a-zA-Z]+)/')
```

Comment:
it's a simple regex, btw an inline doc could be useful, i.e.: `// valid name contains alpha (not numeric) chars`

Reply:
done